src/activity_handlers.cpp

```diff
@@ -1394,6 +1394,9 @@ void activity_handlers::butcher_finish( player_activity *act, Character *you )
 
     you->recoil = MAX_RECOIL;
 
+    get_event_bus().send<event_type::character_butchered_corpse>( you->getID(),
+            corpse_item.get_mtype()->id, ( std::string )act->id() );
```

Comment:
```suggestion
            corpse_item.get_mtype()->id, act->id().str() );
```

It seems akrieger's explanations of how things work were a bit of a distraction of what you should actually do.

Reply:
Sorry ;_;